packages/cms/form/Array/Array.tsx

```diff
@@ -95,6 +98,13 @@ const FormArray: React.FC<FormComponentProps> = ({
                         placeholder={`Add a new ${field.name.toLowerCase()}`}
                         value={newItem}
                         onChange={event => setNewItem(event.target.value)}
+                        onKeyDown = {(
+                            event: KeyboardEvent<HTMLInputElement>
```

Comment:
If you define the type of the onKeyDown, prop in TextInputProps, typescript will know what event is already. So here, you don't need to define what event is. So instead of `event: KeyboardEvent<HTMLInputElement>`, you just need `event`